packages/tailwindcss/src/canonicalize-candidates.test.ts

```diff
@@ -97,7 +97,7 @@ describe.each([['default'], ['with-variant'], ['important'], ['prefix']])('%s',
 
     /// theme(…) to `var(…)`
     // Keep candidates that don't contain `theme(…)` or `theme(…, …)`
-    ['[color:red]', '[color:red]'],
+    ['[color:red]', 'text-[red]'],
```

Comment:
So you will notice that tests changed in this commit. This is because we were testing each migration in isolation before. But now we're testing the entire `canonicalizeCandidates` function at once.

It just happens to be that this is an even _more_ canonicalized compared to before.